<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configure</class>
 <widget class="QDialog" name="Configure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>270</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>config</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_save_config">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>205</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>nPhotons</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_nPhot">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>nLayers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_nLayers">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wavelength</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_wavelength">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>structure</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>40</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_remove">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>160</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>6</y>
       <width>141</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Avalible Materials</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_mat_list"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_invis">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>6</y>
       <width>131</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Build</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_build"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save_struct">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>materials</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>6</y>
       <width>151</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Known Materials</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_known_mats"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_edit_material">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>-10</y>
       <width>20</width>
       <height>251</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_new_mat">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>97</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add new</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>50</y>
       <width>212</width>
       <height>124</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="inputMethodHints">
         <set>Qt::ImhLowercaseOnly</set>
        </property>
        <property name="maxLength">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Kappa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_kappa">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Albedo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_albedo">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Scatter type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_scatter_type"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>270</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_nPhot</tabstop>
  <tabstop>spinBox_nLayers</tabstop>
  <tabstop>spinBox_wavelength</tabstop>
  <tabstop>pushButton_save_config</tabstop>
  <tabstop>listWidget_mat_list</tabstop>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_remove</tabstop>
  <tabstop>listWidget_build</tabstop>
  <tabstop>pushButton_save_struct</tabstop>
  <tabstop>listWidget_known_mats</tabstop>
  <tabstop>pushButton_edit_material</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>doubleSpinBox_kappa</tabstop>
  <tabstop>doubleSpinBox_albedo</tabstop>
  <tabstop>comboBox_scatter_type</tabstop>
  <tabstop>pushButton_new_mat</tabstop>
  <tabstop>pushButton_exit</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_invis</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>doubleSpinBox_albedo</sender>
   <signal>valueChanged(double)</signal>
   <receiver>pushButton_new_mat</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_kappa</sender>
   <signal>valueChanged(double)</signal>
   <receiver>pushButton_new_mat</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_name</sender>
   <signal>editingFinished()</signal>
   <receiver>pushButton_new_mat</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>Configure</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_add</sender>
   <signal>pressed()</signal>
   <receiver>pushButton_save_struct</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_scatter_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pushButton_new_mat</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_remove</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_save_struct</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_wavelength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pushButton_save_config</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Configure</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_nPhot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pushButton_save_config</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_nLayers</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pushButton_save_config</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
